Nextcloud pic and flow update for iOS and macOS (#454)
Start9Labs/documentation: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-mac.rst

Native Desktop Integration
--------------------------

> The smoothest experience will be using direct account integration with your Mac.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps.  The steps below are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/24/user_manual/en/groupware/sync_osx.html>`_ and updated for the latest MacOS (Ventura).  Make sure you have first set up :ref:`LAN access<lan-mac>`.

Reviewer: ```suggestion
The smoothest experience will be using direct account integration with your Mac.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps if you don't have them already (these are installed by default on the latest Nextcloud for StartOS).  The steps below are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/24/user_manual/en/groupware/sync_osx.html>`_ and updated for the latest MacOS (Ventura).  Make sure you have first set up :ref:`LAN access<lan-mac>`.
```

.. tabs::
  
  .. group-tab:: Ventura

    1. Open the "System Settings" and select "Internet Accounts," click "Add Account." and then select "Add Other Account".
    
      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step1.png
        :width: 50%
        :alt: macOS add account
      
      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step1.2.png

>         :width: 50%

Reviewer: ```suggestion
        :width: 40%
```

        :alt: macOS add account

    
    2. Select CalDAV for calendar setup or CardDAV for contacts setup. If you want to do both, you will need to return to this step after finishing the setup of the first.
      
      .. note:: You will need to perform 2 individual setups, one for Calendar and one for Contacts.
      
      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step2.png

>         :width: 50%

Reviewer: ```suggestion
        :width: 40%
```

        :alt: macOS select account
    
    3. Select "Advanced" from the "Account Type" dropdown menu and fill in the following fields:

      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step3.png
        :width: 50%
        :alt: macOS setup account
      
      - Username - The default user is "admin," but this is your user within Nextcloud, so be sure it is the correct user if you have more than one

      - Password - In your Nextcloud WebUI, visit the top-right-hand menu and select "Personal Settings" -> "Security." At the bottom, under Devices & Sessions, create a new app password with a name of your choice, such as "MacCalDAV." Then, copy the resulting password into your Mac's account configuration. 
      
        .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step4.png
          :width: 50%
          :alt: nextcloud app password 

      - Server Address - copy your LAN address from the "Interfaces" section of your Nextcloud service page then paste without `https://` suffix.
      

>       - Server Path - For CalDav enter this path `/remote.php/dav/principals/users/embassy/`. You can find complete path in Nextcloud -> Calendar settings -> Copy iOS/macOS CalDav address. For setting up contacts/CardDav use this path `/remote.php/dav`.

Reviewer: This cannot be right if your nextcloud user name is admin.  I think it's best to just remove the beginning of sentence and lean on the instructions to copy the full path.  Then the second sentence should read:

For Contacts (CardDAV), append `/remote.php/dav` after `.local` to your LAN interface address.

      - Port - Set port for `443`.
  
    4. Click "Sign In."

      - You can now select the apps you want to use on your Mac, such as Calendars, Reminders.

>       - Return to Step 3 to continue setup.

Reviewer: ```suggestion
      - Return to Step 3 to set up CalDAV/CardDAV, whichever you have not done yet.
```

>   .. group-tab:: pre-Ventura

Reviewer: ```suggestion
  .. group-tab:: Pre-Ventura
```

> 1. Download the appropriate desktop client from https://nextcloud.com/install/#install-clients

Reviewer: ```suggestion
1. Download the appropriate desktop client from the `Nextcloud website <https://nextcloud.com/install/#install-clients>`_.
```

> 2. Open the client and click "Log In"

Reviewer: ```suggestion
2. Open the client and click "Log In".
```

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step2.png
    :width: 50%
    :alt: nextcloud-login

> 3. From your server's Nextcloud Service page, go to "Interfaces" and copy the LAN address

Reviewer: ```suggestion
3. From your server's Nextcloud Service page, go to "Interfaces" and copy the LAN address.
```

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step3.png
    :width: 50%
    :alt: nextcloud-login

> 4. Enter your LAN address under "Server Address" and click "Next"

Reviewer: ```suggestion
4. Enter your LAN address under "Server Address" and click "Next".
```

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step4.png
    :width: 50%
    :alt: nextcloud-login

> 5. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards

Reviewer: ```suggestion
5. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards.
```

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step5.png
    :width: 50%
    :alt: nextcloud-login

> 6. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click "Connect"

Reviewer: ```suggestion
6. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click "Connect".
```

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step6.png
    :width: 50%
    :alt: nextcloud-login

7. Files will begin to sync immediately and you will see a green check when this is complete.

  .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step7.png
    :width: 50%
    :alt: nextcloud-login
    

> 8. That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages

Reviewer: ```suggestion
8. That's it! From this desktop client you can receive notifications, control accounts and syncing, and quickly access your apps' WebUI pages.
```

4. Enter your LAN address under "Server Address" and click "Next"

  .. figure:: /_static/images/nextcloud/nextcloud-mac-step4.png
    :width: 40%
    :alt: nextcloud-login

5. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards

  .. figure:: /_static/images/nextcloud/nextcloud-mac-step5.png
    :width: 40%
    :alt: nextcloud-login

6. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click "Connect"

  .. figure:: /_static/images/nextcloud/nextcloud-mac-step6.png
    :width: 60%
    :alt: nextcloud-login

7. Files will begin to sync immediately and you will see a green check when this is complete.

  .. figure:: /_static/images/nextcloud/nextcloud-mac-step7.png
    :width: 50%
    :alt: nextcloud-login
    
8. That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages

Tor Setup
.........
You will first need to have the :ref:`Tor daemon running<tor-mac>`.

> 

Reviewer: Should we specify here that it is best to add a second account for Tor?  What do you think?

Author: I think its ok. Can be confusing. When you change setting in network tab top Tor, LAN is not working. I think most will stay on one account.
